feature/qos/ate_tests/qos_basic_test/qos_basic_test.go

```diff
@@ -1628,34 +1627,32 @@ func ConfigureJuniperQos(t *testing.T, dut *ondatra.DUTDevice) {
 		queueName string
 	}{{
 		desc:      "output-interface-BE1",
-		queueName: "6",
+		queueName: queues.BE1,
 	}, {
 		desc:      "output-interface-BE0",
-		queueName: "0",
+		queueName: queues.BE0,
 	}, {
 		desc:      "output-interface-AF1",
-		queueName: "4",
+		queueName: queues.AF1,
 	}, {
 		desc:      "output-interface-AF2",
-		queueName: "1",
+		queueName: queues.AF2,
 	}, {
 		desc:      "output-interface-AF3",
-		queueName: "5",
+		queueName: queues.AF3,
 	}, {
 		desc:      "output-interface-AF4",
-		queueName: "2",
+		queueName: queues.AF4,
 	}, {
 		desc:      "output-interface-NC1",
-		queueName: "3",
+		queueName: queues.NC1,
 	}}
 
 	t.Logf("qos output interface config: %v", schedulerIntfs)
 	for _, tc := range schedulerIntfs {
 		i := q.GetOrCreateInterface(dp3.Name())
 		i.SetInterfaceId(dp3.Name())
-		if deviations.ExplicitInterfaceRefDefinition(dut) {
-			i.GetOrCreateInterfaceRef().Interface = ygot.String(dp3.Name())
-		}
+		i.GetOrCreateInterfaceRef().Interface = ygot.String(dp3.Name())
```

Comment:
same as above

Reply:
This is under ConfigureJuniperQos, it is specific to Juniper and the mentioned deviation is not applicable